cinq_auditor_vpc_flowlogs/__init__.py

```diff
@@ -85,29 +73,21 @@ def get_iam_role_arn(self, account):
         Returns:
             :obj:`IAMRole`
         """
-        marker = None
         try:
             iam = self.session.client('iam')
-            while True:
-                if marker:
-                    roles = iam.list_roles(Marker=marker)
-                else:
-                    roles = iam.list_roles()
-
-                for item in roles['Roles']:
-                    if item['RoleName'] == 'VpcFlowLogsRole':
-                        return iam.get_role(RoleName='VpcFlowLogsRole')['Role']['Arn']
+            rolearn = iam.get_role(RoleName=self.role_name)['Role']['Arn']
 
-                if roles['IsTruncated']:
-                    marker = roles['Marker']
-                else:
-                    break
+        except ClientError as e:
+            if e.response['Error']['Code'] == 'NoSuchEntity':
+                rolearn = self.create_iam_role(account)
+                pass
+            else:
+                raise
 
-            return None
+        except Exception as e:
+            self.log.exception('Problem confirming the IAM role needed for VPC Flow Log Auditing. {}'.format(e))
 
-        except Exception:
-            self.log.exception(
-                'Problem enumerating roles for account {}. Check the permissions for this account.'.format(account))
+        return rolearn
```

Comment:
This return should be at the bottom of the try block, not outside, since this could cause a possible issue, where it tries to return a variable that might not exist (never return a variable defined in an inner scope, from the outer scope)